### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (539)
Screenshot (540)
Screenshot (541)
Screenshot (542)

### CODE REVIEW

File: Components/Images.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.image-wrapper{
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    max-width: 1000px;
+}
+.responsive-img {
+    max-width: 50%;
+    height: auto;
+  }
```
Comment (hinanshisuthar) [inline Components/Images.css]: Why max-width in 1000px?

File: Components/Images.css [hunk 2]
```diff
@@ -16,7 +16,29 @@
...
-.ul-image .li-image { width: 15%; text-align: center; position: relative; padding-bottom: 15%; border-radius: 50%; overflow: hidden; }
-.ul-image .li-image .img-component { position: absolute; width: 100%; height: 100%; top: 0; left: 0; object-fit: cover; }
\ No newline at end of file
+.ul-image {
+    display: -webkit-flex;
+    display: flex;
+    text-align: center;
+    justify-content: space-between;
+    margin-bottom: 30px;
+}
```
Comment (stutisk) [inline Components/Images.css:26]: Use rem instead of px

**Review by hinanshisuthar — COMMENTED** [review]
Looks good. Add a new line at the EOF for good practices.

File: Docsite-HTML-Files/Toast.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
     <title>Toast Component</title>
     <link rel="stylesheet" href="/Components/Villainess.css" />
     <script src="https://kit.fontawesome.com/c9ffd32afb.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/79dfc2e9eb.js" crossorigin="anonymous"></script>
```
Comment (Hamdrive) [inline Docsite-HTML-Files/Toast.html:11]: Use the CDN link of FontAwesome instead

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Changes have been suggested, else code looks good and proper practices followed!

**Review by stutisk — APPROVED** [review]

**Pull request merged**